src/js/msp/msp_queue_monitor.js

```
@@ -0,0 +1,692 @@
+        }
+        if (this.originalDispatchMessage) {
+            this.msp._dispatch_message = this.originalDispatchMessage;
+        }
+        if (this.originalRemoveRequest) {
+            this.msp._removeRequestFromCallbacks = this.originalRemoveRequest;
+        }
+
+        // Clear instrumentation flag
+        delete this.msp._mspQueueMonitorInstrumented;
+
+        this.listeners = [];
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)